source/_redirects

```diff
@@ -218,7 +218,7 @@
 /components/camera.doorbird /components/doorbird#camera
 /components/camera.familyhub /components/familyhub
 /components/camera.foscam /components/foscam
-/components/camera.generic /components/generic
+/components/camera.generic /components/generic_ip_camera
```

Comment:
Thanks, Franck.
I thought just updating the redirects file for _/components/camera.generic_ should've been enough.
Or was I supposed to add a new redirect entry from _/components/generic_ to _/components/generic_ip_camera_ instead?

Reply:
Both! 😄

Reply:
Thank you! I hope I've got it right this time.

(Apologies for any misunderstanding - first PR ever, very nervous 😄)

Reply:
Don't be nervous! We are all people and we all had the same experience once.
So don't worry! It always makes me happy to see somebody step up and joining the open-source game, that sir, deservice a medal 🏅

Reply:
Franck, thanks a lot!🤗 It means a great deal for me as a new-comer. I'm so glad to get introduced to the open-source community via this project. 

P.S. I'll save and print out this PR for getting back to it at any time of self-doubt 🙃